MapleServer2/PacketHandlers/Game/ClubHandler.cs

```diff
@@ -57,25 +78,53 @@ public override void Handle(GameSession session, PacketReader packet)
 
     private static void HandleCreate(GameSession session, PacketReader packet)
     {
-        // TODO fix creating for a party of more than 2. Currently if a member does not respond, despite atleast one other member accepting, it does not get created.
         Party party = GameServer.PartyManager.GetPartyByLeader(session.Player);
-        if (party == null)
+        if (party is null || party.Leader != session.Player)
```

Comment:
I think comparing objects is not the best idea, maybe it could desync, I would compare character ids